### DISCUSSION TITLE
IPV6 in T-POT

### DISCUSSION THREAD

**Discussion by sonzu-prog** [telekom-security/tpotce#1455] [Q&A]
Hello, good day to all. Is it ok to disable ipv6 on the host where TPOT is installed?

**Comment by Ganesh-403** [comment]
@sonzu-prog 
Yes, **it's generally OK to disable IPv6 on the T-Pot host if your environment only uses IPv4**.

From what I can tell, T-Pot does **not** require IPv6 to function. Its honeypots, Docker containers, and management services work over IPv4, and there is no documented requirement in the T-Pot installation guide that IPv6 must be enabled.

That said, there are a few considerations:

- If your network **doesn't use IPv6**, disabling it can simplify the networking stack and reduce unnecessary exposure.
- If you **do** have IPv6 connectivity (or plan to monitor IPv6 traffic), you should leave it enabled. Docker supports IPv6 networking, but it must be explicitly configured on the Docker daemon and networks. :contentReference[oaicite:0]{index=0}

If you decide to disable IPv6, make sure it's disabled consistently on the host (rather than partially disabled on some interfaces), as Linux treats IPv6 as a kernel networking feature. :contentReference[oaicite:1]{index=1}

One thing to verify after making the change is that Docker and T-Pot still start correctly:

```bash
sudo systemctl restart docker
sudo systemctl restart tpot

docker ps
```

If all containers come up normally and the Web UI is accessible, then disabling IPv6 hasn't negatively affected your deployment.

One question:

- Is your T-Pot instance **Internet-facing**, or is it deployed only on an internal IPv4 network? If it's Internet-facing and your ISP provides IPv6, you may want to keep IPv6 enabled so the honeypots can also observe IPv6-based scanning and attacks.

If this solves your problem, feel free to mark it as the accepted answer so others can find it easily.
